packages/terra-dialog/CHANGELOG.md

```diff
@@ -5,4 +5,4 @@ Unreleased
 -----------------
 * Initial stable release
 * New component, examples, doc, tests.
-* Added theme variables.
```

Comment:
Update the changelog.

Reply:
As an unreleased component, I'm not sure that a changelog makes much sense. The only reason anything is here at all is because DangerJS is mean.